narayana-bom/pom.xml

```diff
@@ -929,6 +665,11 @@
         <artifactId>wrapper</artifactId>
         <version>${version.tanukisoft}</version>
       </dependency>
+      <dependency>
+        <groupId>io.quarkus</groupId>
+        <artifactId>quarkus-resteasy</artifactId>
+        <version>${version.quarkus}</version>
+      </dependency>
```

Comment:
It was added because it is present in Narayana (https://github.com/jbosstm/narayana/blob/main/rts/at/tx/pom.xml#L162) but it has not been included in the bom yet.